commands/about.js

```diff
@@ -0,0 +1,30 @@
+// console logging
+const namespace = 'Slash';

```

Comment:
I suggest a clearer name like `loggerNamespace`.

Reply:
Called it `logTag`!